<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtandLeisaiClass</class>
 <widget class="QMainWindow" name="QtandLeisaiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtandLeisai</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
border:2px groove gray;border-radius:10px;padding:2px 4px;
	font: 14pt &quot;新宋体&quot;;
}                          
QPushButton:hover{
background-color:rgb(229, 241, 251); 
color: black;}
QPushButton:pressed{
background-color:rgb(204, 228, 247);
border-style: inset;}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
    </property>
    <property name="title">
     <string>初始化</string>
    </property>
    <widget class="QPushButton" name="Bt_AxisGoHome">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>回原点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_AxisEn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>轴使能</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_CardLint">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_AxisRestEn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>复位使能</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_AixsSta">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>伺服状态：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_CleanAisxWarn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清除伺服报警</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_CardClose">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_RunTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>启动Time</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
    </property>
    <property name="title">
     <string>轴控制</string>
    </property>
    <widget class="QPushButton" name="Bt_AxisJOGSub">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>JOG-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_AxisJOGAdd">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>JOG+</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_MovePS">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>目标位置：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>当前位置：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_AxisRunAbs">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>绝对定位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_AxisStop">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>速度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_AxisSpeed">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>20000</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chB_MoveOri">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>反向</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_AxisPosition">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Bt_AxisRunRel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>相对定位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>当前状态：</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_Aixs_Move_Sta">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>JOG移动量：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_JogMoveVlue">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>20000</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
    </property>
    <property name="title">
     <string>插补</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>901</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
    </property>
    <property name="title">
     <string>I/O监视</string>
    </property>
    <widget class="QTableWidget" name="TK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>881</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="LE_OutPutVule">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>350</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>输出口：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_SetOut">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_RestOut">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rest</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtandLeisai.qrc"/>
 </resources>
 <connections/>
</ui>
